<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="mainGV"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="action_model_vs_model"/>
    <addaction name="action_user_X_vs_model_O"/>
    <addaction name="action_model_X_vs_user_O"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_model_vs_model">
   <property name="text">
    <string>model vs model</string>
   </property>
  </action>
  <action name="action_user_X_vs_model_O">
   <property name="text">
    <string>user (X) vs model (O)</string>
   </property>
  </action>
  <action name="action_model_X_vs_user_O">
   <property name="text">
    <string>model (X) vs user (O)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
